data/lib_sram_data.c

```diff
@@ -0,0 +1,34 @@
+#include "global.h"
+#include "src/lib_sram.h"
+
+extern u16 sVerifySramFastWorkCopy[];
+extern u16 sReadSramFastWorkCopy[];
+extern u32 sWriteIntSramFastWorkCopy[];
+
+extern void (*write_int_sram_fast)(const u8 *src, u8 *dest, u32 size);
+extern const s32 write_int_sram_fast_end;
+
+// SRAM Library Version
+const char lib_sram_version[] = "SRAM_F_V110";
+
+const void *const lib_sram_pointers[] = {
+    (void *)&gWriteIntSramFast,
```

Comment:
what is going on here lmao, are these void pointer casts necessary for no warnings? if so then ig nothing we can do but man is it ugly

Reply:
Yeah, it's required and it looks awful. Good thing every bit of data in that file is completely unused.